src/common/error_log.cpp

```diff
@@ -115,5 +118,255 @@ void FFDCFile::removeFFDCFile()
     std::remove(_fileName.c_str());
 }
 
+FFDCFiles::FFDCFiles(const bool collectTraces, const json& calloutsDetails)
+{
+    // Create FFDCFile for the traces if requested to collect
+    if (collectTraces)
+    {
+        try
+        {
+            createFFDCFileForTraces();
+        }
+        catch (const std::exception& e)
+        {
+            /**
+             * Don't throw the exception, we should create FFDCFiles as much as
+             * possible to create the error log.
+             */
+            log<level::ERR>(fmt::format("Exception [{}]", e.what()).c_str());
```

Comment:
Better to update text with  menigfull trace , no actions related to this . otherwise it si going mislead during debug

Reply:
Done